### PR TITLE
OTG-152 remove Webscarab references

### PR DESCRIPTION
**Pull request by mackowski**
This PR covers issue #152.

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- remove Webscarab references across testing guide

### CODE REVIEW

**Comment by ThunderSon** [comment]
@kingthorin do you think we should have a replacement in place, which is ZAP in the case of WebScarab? Or should we set this as part of updating every testing scenario?

**Comment by kingthorin** [comment]
I think it definitely makes sense to replace, and since ZAP is also an OWASP project it's the logical choice.

**Comment by mackowski** [comment]
Sure I will update this PR by adding ZAP

**Comment by kingthorin** [comment]
Thanks @mackowski keep in mind there are places where the webscarab removal make sense and zap already exists, like: https://github.com/OWASP/OWASP-Testing-Guide-v5/pull/181/files#diff-bd152f06cf3a06404610ad7af3323b2cL114-R115

Also instances like: https://github.com/OWASP/OWASP-Testing-Guide-v5/pull/181/files#diff-c947191939e134293f9080966df37993R17 should probably be `a web proxy`

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ The following are some common tool types that can be useful in identifying busin
 
 **Intercepting Proxy - To observe the request and response blocks of HTTP traffic.**\
 
+-   OWASP ZAP - <https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project>
+
+<!-- -->
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Please drop this comment, and the additional blank line.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 2]
```diff
@@ -88,6 +88,9 @@ The following are some common tool types that can be useful in identifying busin
 
 **Intercepting Proxy - To observe the request and response blocks of HTTP traffic.**\
 
+-   OWASP ZAP - <https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project>
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Using properly formatted markdown links is preferred:

`[OWASP Zed Attack Proxy (ZAP)](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project)`

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.9_Testing_for_SSI_Injection_OTG-INPVAL-009.md [hunk 3]
```diff
@@ -75,6 +75,7 @@ Tools
 -----
 
 -   Web Proxy Burp Suite - <http://portswigger.net>
+-   OWASP Zed Attack Proxy (ZAP) - <https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project>
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.9_Testing_for_SSI_Injection_OTG-INPVAL-009.md]: Using properly formatted markdown links is preferred:

`[OWASP Zed Attack Proxy (ZAP)](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project)`

**Comment by mackowski** [comment]
I replaced Webscarab to ZAP when it made sense

File: document/4_Web_Application_Security_Testing/4.4_Identity_Management_Testing/4.4.4_Testing_for_Account_Enumeration_and_Guessable_User_Account_OTG-IDENT-004.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ In black box testing, the tester knows nothing about the specific application, u
 
 Record the server answer when you submit a valid user ID and valid password.
 
-> Using WebScarab, notice the information retrieved from this successful authentication (HTTP 200 Response, length of the response).
+> Using web proxy, notice the information retrieved from this successful authentication (HTTP 200 Response, length of the response).
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.4_Identity_Management_Testing/4.4.4_Testing_for_Account_Enumeration_and_Guessable_User_Account_OTG-IDENT-004.md]: `a web proxy`

File: document/4_Web_Application_Security_Testing/4.4_Identity_Management_Testing/4.4.4_Testing_for_Account_Enumeration_and_Guessable_User_Account_OTG-IDENT-004.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ Now, the tester should try to insert a valid user ID and a wrong password and re
 >
 > `Login for User foo: invalid password`
 >
-> Using WebScarab, notice the information retrieved from this unsuccessful authentication attempt (HTTP 200 Response, length of the response).
+> Using web proxy, notice the information retrieved from this unsuccessful authentication attempt (HTTP 200 Response, length of the response).
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.4_Identity_Management_Testing/4.4.4_Testing_for_Account_Enumeration_and_Guessable_User_Account_OTG-IDENT-004.md]: `a web proxy`

File: document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.1_Testing_for_Credentials_Transported_over_an_Encrypted_Channel_OTG-AUTHN-001.md [hunk 6]
```diff
@@ -14,11 +14,11 @@ Nowadays, the most common example of this issue is the log in page of a web appl
 
 ### Black Box Testing
 
-In the following examples we will use WebScarab in order to capture packet headers and to inspect them. You can use any web proxy that you prefer.
+In the following examples we will use web proxy in order to capture packet headers and to inspect them. You can use any web proxy that you prefer.
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.1_Testing_for_Credentials_Transported_over_an_Encrypted_Channel_OTG-AUTHN-001.md]: `a web proxy`

File: document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.1_Testing_for_Credentials_Transported_over_an_Encrypted_Channel_OTG-AUTHN-001.md [hunk 7]
```diff
@@ -14,11 +14,11 @@ Nowadays, the most common example of this issue is the log in page of a web appl
...
 
 #### Example 1: Sending Data with POST Method Through HTTP
 
-Suppose that the login page presents a form with fields User, Pass, and the Submit button to authenticate and give access to the application. If we look at the headers of our request with WebScarab, we can get something like this:
+Suppose that the login page presents a form with fields User, Pass, and the Submit button to authenticate and give access to the application. If we look at the headers of our request with web proxy, we can get something like this:
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.1_Testing_for_Credentials_Transported_over_an_Encrypted_Channel_OTG-AUTHN-001.md]: `a web proxy`

File: document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.6_Testing_for_Browser_Cache_Weaknesses_OTG-AUTHN-006.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ The “Back” button can be stopped from showing sensitive data. This can be do
 
 ### Browser Cache
 
-Here testers check that the application does not leak any sensitive data into the browser cache. In order to do that, they can use a proxy (such as WebScarab) and search through the server responses that belong to the session, checking that for every page that contains sensitive information the server instructed the browser not to cache any data. Such a directive can be issued in the HTTP response headers:
+Here testers check that the application does not leak any sensitive data into the browser cache. In order to do that, they can use a proxy (such as OWSAP ZAP) and search through the server responses that belong to the session, checking that for every page that contains sensitive information the server instructed the browser not to cache any data. Such a directive can be issued in the HTTP response headers:
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.5_Authentication_Testing/4.5.6_Testing_for_Browser_Cache_Weaknesses_OTG-AUTHN-006.md]: `OWASP`

**Comment by mackowski** [comment]
Thanks for review. 
If you want I can change all links (not only ZAP ones) from these files to use proper markdown links instead of <>

**Comment by kingthorin** [comment]
If you feel like changing them all that would be great!

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 9]
```diff
@@ -83,157 +83,109 @@ While there are tools for testing and verifying that business processes are func
...
+**Web Browser Plug-ins - To view and modify HTTP/HTTPS headers, post parameters and observe the DOM of the Browser**
 
--   Paros Proxy - <http://www.parosproxy.org/>
+-   [Tamper Data (for Internet Explorer)](https://addons.mozilla.org/en-us/firefox/addon/tamper-data/)
 
-**Web Browser Plug-ins - To view and modify HTTP/HTTPS headers, post parameters and observe the DOM of the Browser**\
+-   [TamperIE (for Internet Explorer)](http://www.bayden.com/tamperie/)
 
--   Tamper Data (for Internet Explorer) - <https://addons.mozilla.org/en-us/firefox/addon/tamper-data/>
+-   [Firebug (for Internet Explorer)](https://addons.mozilla.org/en-us/firefox/addon/firebug/)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Remove `(for internet explorer)`

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 10]
```diff
@@ -83,157 +83,109 @@ While there are tools for testing and verifying that business processes are func
...
-<!-- -->
+-   [Burp Proxy](http://portswigger.net/burp/proxy.html)
 
--   Burp Proxy - <http://portswigger.net/burp/proxy.html>
 
-<!-- -->
+**Web Browser Plug-ins - To view and modify HTTP/HTTPS headers, post parameters and observe the DOM of the Browser**
 
--   Paros Proxy - <http://www.parosproxy.org/>
+-   [Tamper Data (for Internet Explorer)](https://addons.mozilla.org/en-us/firefox/addon/tamper-data/)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Remove `(for internet explorer)`

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 11]
```diff
@@ -83,157 +83,109 @@ While there are tools for testing and verifying that business processes are func
...
 
-**Web Browser Plug-ins - To view and modify HTTP/HTTPS headers, post parameters and observe the DOM of the Browser**\
+-   [TamperIE (for Internet Explorer)](http://www.bayden.com/tamperie/)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: @kingthorin let's remove this. IE is so old I'd prefer to stay off of it. It's not even Edge.

**Comment by ThunderSon** [comment]
@mackowski great job. You even went further. Small changes and we can merge.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 9 above)
Comment (kingthorin) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Good catch

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 11 above)
Comment (kingthorin) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Agreed.

**Comment by kingthorin** [comment]
@mackowski let us know if you need assistance addressing the conflicts and remaining feedback.

**Comment by mackowski** [comment]
@kingthorin I just updated it :-)

**Pull request merged**